components/tag-list/tag-list.js

```diff
@@ -347,15 +341,9 @@ class TagList extends LocalizeCoreElement(InteractiveMixin(ArrowKeysMixin(LitEle
 		));
 
 		announce(this.localize('components.tag-list.cleared-all'));
-
```

Comment:
This is no longer needed, since as mentioned previously, it wasn't doing anything.